### ISSUE TITLE
NavBar-Button for shell script not available

### ISSUE THREAD

**Issue by eloo** [vitormhenrique/OctoPrint-Enclosure#423]
**Describe the bug**
Hi,
i have configured a shell script as output and activated the "Show Button on Navbar" option.
But i cant find the button in the navbar. 

**To Reproduce**
Create a shell output like this:
![image](https://user-images.githubusercontent.com/2631895/120941711-6dcf8a00-c724-11eb-93e3-3140e9cda80c.png)

**Expected behavior**
I would expect some kind of a button in the navbar. But i can only see the button in the enclosure plugin tab.

**Actual behavior**
![image](https://user-images.githubusercontent.com/2631895/120941733-99527480-c724-11eb-9a33-36d3b4ca430c.png)

**Additional context**
OctoPrint 1.6.1 
Python 3.7.3 
OctoPi 0.18.0
Enclosure Plugin 4.13.1

**Comment by michaelbeljaars** [comment]
Yeah, I have the same thing. I am using two shell scripts to turn on and off a LED strip. I tried enabling the add to NavBar functionality for easier access, but nothing happens.

**Comment by zach-ovic** [comment]
Same here. But it looks like nobody is maintaining this anymore, right? :(

**Comment by michaelbeljaars** [comment]
> Same here. But it looks like nobody is maintaining this anymore, right? :(

Looks like it indeed. It's too bad, I haven't found an other plugin that has the same functionality.
